<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup_Window</class>
 <widget class="QMainWindow" name="signup_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>signup_Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="main_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/home/boi/Projects/C++/Uni/Pos/img/bg.png);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/bg_image/pos_bg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_welcome">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(176, 240, 241);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Segoe UI'; font-size:36pt; font-weight:700; color:#008a8a;&quot;&gt;Welcome&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>361</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border:none;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_signup">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>320</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 148, 148);
font: 700 12pt &quot;Segoe UI&quot;;
border: none;
border-top-left-radius :10px;
border-top-right-radius : 10px; 
border-bottom-left-radius : 10px; 
border-bottom-right-radius : 10px</string>
     </property>
     <property name="text">
      <string>Sign up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 700 11pt &quot;Segoe UI&quot;;
border:none;
border-top-left-radius :10px;
border-top-right-radius : 10px; 
border-bottom-left-radius : 10px; 
border-bottom-right-radius : 10px</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>269</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
color: rgb(88, 88, 88);</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>269</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
color: rgb(88, 88, 88);</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_email">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>269</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;
font: 14pt &quot;Segoe UI&quot;;
color: rgb(89, 89, 89);</string>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_employee">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 95, 95);
font: 700 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Employee</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_admin">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 95, 95);
font: 700 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>269</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;
background-color: rgb(215, 255, 254);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>269</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;
background-color: rgb(215, 255, 254);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>269</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(189, 255, 255);
color: rgb(0, 0, 0);
font: 13pt &quot;Segoe UI&quot;;
background-color: rgb(193, 255, 253);
background-color: rgb(215, 255, 254);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>[+]DB status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
